shepherd.js/src/utils/floating-ui.ts

```diff
@@ -211,8 +211,13 @@ export function getFloatingUIOptions(
     );
 
     if (arrowEl) {
+      const arrowOptions =
+        typeof step.options.arrow === "object"
+        ? step.options.arrow
+        : { padding: 4 };
+
       options.middleware.push(
-        arrow({ element: arrowEl, padding: hasEdgeAlignment ? 4 : 0 })
+        arrow({ element: arrowEl, padding: hasEdgeAlignment ? arrowOptions.padding : 0 })
```

Comment:
```suggestion
        arrow({ element: arrowEl, padding: hasEdgeAlignment ? step.options.arrow?.padding ?? 4 : 0 })
```

What would you think about putting this inline here? I think it is a little less verbose.